x-pack/test/plugin_api_integration/plugins/task_manager/init_routes.js

```diff
@@ -24,6 +24,14 @@ const taskManagerQuery = {
 };
 
 export function initRoutes(server, taskManager, legacyTaskManager, taskTestingEvents) {
+  const callCluster = server.plugins.elasticsearch.getCluster('admin').callWithInternalUser;
+
+  async function ensureIndexIsRefreshed() {
```

Comment:
hahahahaha

Reply:
You laugh, I spent a whole morning trying to figure out WTH was going on 😆